tools/configure/cmdArgs.ml

```diff
@@ -143,6 +147,8 @@ let check_absolute = function
 let args_options = Arg.align [
   "-prefix", arg_string_option (fun p prefix -> check_absolute prefix; { p with prefix }),
     "<dir> Set installation directory to <dir> (absolute path required)";
+  "-local", arg_set (fun p -> local p),
+    " Set prefix to \""^local_prefix^"\" and disable installation";
```

Comment:
For sure I don't want -local to be back, what's the need?

Reply:
People keep asking for it

Reply:
https://coq.zulipchat.com/#narrow/stream/237656-Coq-devs.20.26.20plugin.20devs/topic/.2E.2Fconfigure.20-local.20no.20longer.20exists.2C.20what.20to.20use.20now.3F/near/269321868

Reply:
That's not people asking for it, they were installing from sources in a weird way; they can just use a supported method actually. `-local` was all about having a non-standard layout, which goes against the direction all modern build systems take.

Reply:
GNU `stow` has existed for decades for that use case, for example.

Reply:
>That's not people asking for it, they were installing from sources in a weird way

I don't understand what you mean, the answer they got was to use profile devel, which is just local plus some other stuff. What's wrong with exposing local when we already have it as part of devel?

Reply:
Actually the solution to use `-profile devel` is also wrong, that's not a release build and will not be optimized properly.

Reply:
What is wrong is that you are encouraging people to use an option that IMHO is wrong for their use case.

Reply:
How do you know their use case? And what's wrong with local?

Reply:
I assumed their use case is to actually use Coq, not to develop Coq. If so, the right answer is to use a release build.

Reply:
@robbertkrebbers what's your use case?

Reply:
> I don't understand what you mean, the answer they got was to use profile devel, which is just local plus some other stuff. What's wrong with exposing local when we already have it as part of devel?

That's the answer you give them, but I already complained about this answer in https://coq.discourse.group/t/how-to-use-a-local-installation-of-coq/1507.

Reply:
That's just restating that local was removed, ie solved by ths PR